cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -58,7 +63,7 @@ public final class InitCommand extends BaseCommand {
         super(commonOptions, initOptions.helidonVersion() != null);
         this.commonOptions = commonOptions;
         this.initOptions = initOptions;
-        this.batch = initOptions.batch();
+        this.batch = initOptions.batch() || initOptions.propsFileOption() != null;
```

Comment:
Do we have to infer batch when `--props-file` is supplied ? It should also work in interactive right ?